telemetry/definitions/commonDefinitions.json

```diff
@@ -3797,6 +3797,10 @@
                 {
                     "type": "codeTransformJobId",
                     "required": false
+                },
```

Comment:
if this is `required:true` all existing implementation will need update to populate this field.

Reply:
That's fine. We want to update them to include this field.